WordPress/src/main/java/org/wordpress/android/ui/menus/items/MenuItemEditorFactory.java

```diff
@@ -18,8 +18,44 @@
         TAG,
         LINK,
         JETPACK_TESTIMONIAL,
+        JETPACK_PORTFOLIO,
+        JETPACK_COMIC,
         CUSTOM;
 
+        public static String stringForType(ITEM_TYPE type) {
```

Comment:
The intention of this method seems to conflict with `nameForItemType` below, plus it's not being used.